app/Services/Tmdb/Client/Movie.php

```diff
@@ -246,7 +263,7 @@ public function __construct(int $id)
             ->get('https://api.TheMovieDB.org/3/movie/{id}', [
                 'api_key'            => config('api-keys.tmdb'),
                 'language'           => config('app.meta_locale'),
-                'append_to_response' => 'videos,images,credits,external_ids,keywords,recommendations,alternative_titles',
+                'append_to_response' => 'videos,images,credits,external_ids,keywords,recommendations,alternative_titles,release_dates,content_ratings',
```

Comment:
`content_ratings` is not a valid value for movies. It's only a valid value for tv.